azure_ad_admin/plugin.spec.yaml

```
@@ -545,6 +545,23 @@ actions:
         description: Was operation successful
         type: boolean
         required: true
+  revoke_sign_in_sessions:
+    title: Revoke Sign-In Sessions
+    description: Invalidates all the refresh tokens issued to applications for a user (as well as session cookies in a user's browser), by resetting the signInSessionsValidFromDateTime user property to the current date-time
```

This seems a bit too technical, especially compared to the other action descriptions.  
```suggestion
    description: Logs the user out of all AD Azure applications.  The user must re-login to all applications.
```